backend/azureblob/azureblob.go

```diff
@@ -217,6 +217,20 @@ This option controls how often unused buffers will be removed from the pool.`,
 				encoder.EncodeDel |
 				encoder.EncodeBackSlash |
 				encoder.EncodeRightPeriod),
+		}, {
+			Name:    "public_access",
+			Help:    "Public access level of a container: blob, container.",
+			Default: string(azblob.PublicAccessNone),
+			Examples: []fs.OptionExample{
+				{
```

Comment:
This needs a "none" branch for no public access which should be the first option and the default.

Reply:
Since the default value is `''` ("None" branch resolves to empty string i.e `''`) hence wonder if this does sound okay? Please advise.

```
{
       Value: "",
	Help:  "The container and its blobs can be accessed only with an authorized request. This is the default value.",
}

```

Reply:
I think using the `string(azblob.PublicAccessNone)` names is fine.